forward_engineering/helpers/alterScriptHelpers/alterEntityHelper.js

```diff
@@ -45,20 +67,29 @@ module.exports = (app, options) => {
 			jsonSchema,
 			idToNameHashTable,
 		});
-		const tableScript = ddlProvider.createTable(hydratedTable, jsonSchema.isActivated);
+		const tableScriptDto = AlterScriptDto.getInstance([ddlProvider.createTable(hydratedTable, jsonSchema.isActivated)], true, false);
+		const indexesScriptsDto = AlterScriptDto.getInstances(indexesScripts, true, false);
 
-		return [tableScript, ...indexesScripts].join('\n\n');
+		return [tableScriptDto, ...indexesScriptsDto];
```

Comment:
Just to be safe, when you return an array of AlterScriptDTOs, add ".filter(Boolean)" at the end, as factory methods may return undefined